content/tbd/1000 Landscapes for 1 billion people.md

```diff
@@ -0,0 +1,53 @@
+---
+created: 2023-11-30
+contributors:
+  - Catherine Tran
+title: 1000 Landscapes for 1 Billion People
+url: https://landscapes.global/
+locations:
+  - International network
+start: 2018
+end: 
+size: 4-10
+image: assets/img/1000-landscapes-for-1-billion-people-homepage.png
```

Comment:
for this we do need a leading `/` i.e.

```
/assets/img/1000-landscapes-for-1-billion-people-homepage.png
```

Personally i think the best here would be to start to use obsidian links as easiest, least error prone approach ie. start typing ...

`[[...]]`